src/cf-forms/usage.md

```diff
@@ -130,26 +131,54 @@ Inputs should always be paired with a `label` for accessibility reasons.
 
 ### Basic text inputs
 
-<label class="a-label" for="text-input-example">A text input</label>
+<label class="a-label" for="textinput-example">A text input</label>
```

Comment:
Yes, I realized the `id` on the text area was `textarea` so I updated the inputs to match that example.